web/packages/teleterm/src/ui/components/ProfileStatusError.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Teleport
+ * Copyright (C) 2024 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Warning } from 'design/Icon';
+import { P3, Flex } from 'design';
+
+export function ProfileStatusError(props: {
+  error: string;
+  mb?: number | string;
+}) {
+  return (
+    <Flex mb={props.mb}>
+      <Warning color="error.main" size="small" mr={1} />
+      <P3>{toWellFormattedConnectionError(props.error)}</P3>
+    </Flex>
+  );
+}
+
+/** Takes the last line from the error and capitalizes it. */
```

Comment:
are we using this comment format now even for single lines? If so, I'll change my patterns. it probably doesn't matter tbh

Reply:
It'a a JSDoc comment format:
>JSDoc comments should generally be placed immediately before the code being documented. Each comment must start with a /** sequence in order to be recognized by the JSDoc parser. Comments beginning with /*, /***, or more than 3 stars will be ignored. This is a feature to allow you to suppress parsing of comment blocks.

https://jsdoc.app/about-getting-started